src/diffusers/utils/testing_utils.py

```diff
@@ -132,6 +132,33 @@ def numpy_cosine_similarity_distance(a, b):
     return distance
 
 
+def check_if_dicts_are_equal(dict1, dict2):
+    for key, value in dict1.items():
```

Comment:
Maybe add: `dict1, dict2 = dict1.copy(), dict2.copy()` to avoid mutating the dicts, which could make debugging of the tests harder.